recipe/meta.yaml

```diff
@@ -16,6 +16,8 @@ build:
   number: 4
   run_exports:
     - {{ pin_subpackage('dispenso', max_pin='x.x') }}
+  ignore_run_exports:
+    - libstdcxx-ng  # [linux]
```

Comment:
It makes no sense to ignore the run-export and then re-add it manually. What problem were you trying to solve here? In any case, this is not the right approach and should be reverted - happy to help fix whatever it is that needed fixing in a better way.

Reply:
It was to resolve this error ([log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1013057&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=2491)):

```
Warning: rpath /home/conda/feedstock_root/build_artifacts/dispenso_1724896726150/_build_env/lib is outside prefix /home/conda/feedstock_root/build_artifacts/dispenso_1724896726150/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl (removing it)
   INFO: sysroot: '/home/conda/feedstock_root/build_artifacts/dispenso_1724896726150/_build_env/x86_64-conda-linux-gnu/sysroot/' files: '['usr/share/zoneinfo', 'usr/share/locale/zh_TW/LC_MESSAGES/libc.mo', 'usr/share/locale/zh_CN/LC_MESSAGES/libc.mo', 'usr/share/locale/vi/LC_MESSAGES/libc.mo']'
  ERROR (dispenso,lib/libdispenso.so.1.3.0): Needed DSO lib/libstdc++.so.6 found in ['conda-forge/linux-64::libstdcxx==14.1.0=hc0a3c3a_1']
  ERROR (dispenso,lib/libdispenso.so.1.3.0): .. but ['conda-forge/linux-64::libstdcxx==14.1.0=hc0a3c3a_1'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
  ERROR (dispenso,lib/libdispenso.so.1.3.0): Needed DSO lib/libgcc_s.so.1 found in ['conda-forge/linux-64::libgcc==14.1.0=h77fa898_1']
  ERROR (dispenso,lib/libdispenso.so.1.3.0): .. but ['conda-forge/linux-64::libgcc==14.1.0=h77fa898_1'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (dispenso,lib/libdispenso.so.1.3.0): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_16
   INFO (dispenso,lib/libdispenso.so.1.3.0): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_16
   INFO (dispenso,lib/libdispenso.so.1.3.0): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/ld-linux-x86-64.so.2 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_16
WARNING (dispenso): run-exports library package conda-forge/linux-64::libstdcxx-ng==14.1.0=h4852527_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
Traceback (most recent call last):
```

The build errors still look mysterious to me. Any help would be appreciated!

Reply:
https://github.com/conda-forge/dispenso-feedstock/pull/7